common/source/docs/common-baro-external.rst

```diff
@@ -15,7 +18,8 @@ Most autopilots include a barometer directly on the autopilot but attaching an e
 Where to Buy
 ------------
 
-- `MS5611 from Drotek <https://drotek.com/shop/en/imu/44-ms5611-pressure-barometric-board.html>`__ (recommended)
+- `Beyond Robotix Air Data Module (AUAV) <https://www.beyondrobotix.com/products/air-data-module>`_
```

Comment:
this should in a DroneCan catergory, not with the i2c sensors...

Reply:
Ah ok, so this external barometer page is for i2c sensors only? it would be quite good to have a general page about external barometers ... but if not I can remove it